plugins/notifications-backend/src/service/handlers.ts

```diff
@@ -30,31 +30,23 @@ export async function createNotification(
   let isUser = false;
 
   // validate users
-  if (Array.isArray(req.targetGroups) && req.targetGroups.length > 0) {
+  if (
+    req.targetGroups &&
```

Comment:
I think we do not need this check since it is included in the `Array.isArray()`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
Or is it just to make the TypeScript happy?